pkg/cmd/server/bootstrappolicy/securitycontextconstraints_test.go

```diff
@@ -2,34 +2,46 @@ package bootstrappolicy
 
 import (
 	"reflect"
+	"sort"
 	"testing"
 
 	"k8s.io/apiserver/pkg/authentication/serviceaccount"
 
 	securityapi "github.com/openshift/origin/pkg/security/apis/security"
+	scc "github.com/openshift/origin/pkg/security/securitycontextconstraints"
 )
 
 func TestBootstrappedConstraints(t *testing.T) {
-	expectedConstraints := []string{
-		SecurityContextConstraintPrivileged,
+	// ordering of expectedConstraintNames is important, we check it against scc.ByPriority
+	expectedConstraintNames := []string{
+		SecurityContextConstraintsAnyUID,
+		SecurityContextConstraintsHostNetwork,
 		SecurityContextConstraintRestricted,
 		SecurityContextConstraintNonRoot,
-		SecurityContextConstraintHostMountAndAnyUID,
 		SecurityContextConstraintHostNS,
-		SecurityContextConstraintsAnyUID,
-		SecurityContextConstraintsHostNetwork,
+		SecurityContextConstraintHostMountAndAnyUID,
+		SecurityContextConstraintPrivileged,
```

Comment:
this looks like a bug.  The host namespaces should be involved in the values. I'm betting they have matching point value and no priority so they're sorting by name.